frontend/src/app/shared/components/autocompleter/op-autocompleter/op-autocompleter.component.html

```diff
@@ -118,9 +118,9 @@
 
 <ng-template let-item let-search="search"  #defaultOption [ngSwitch]="true">
   <ng-container *ngSwitchCase="resource === 'work_packages' || resource === 'parent-child'">
-    <div class="op-autocompleter--option-wrapper">
+    <div class="op-autocompleter--option-wp-wrapper">
       <op-principal
-        class="op-autocompleter--option-principal"
+        class="op-autocompleter--wp-principal"
```

Comment:
The autocompleter is a shared component which can be used for other resources than work packages as well. So I'd prefer to not use `wp` in the name.

Reply:
Hi @HDinger There are some default options for op-autocompleter like workapackage or user. For these cases, is it good to use class names like op-autocompleter--option-wp or op-autocompleter--option-principal? For example for workpackage option, if there is a principal inside it, we can name its class like op-autocompleter--option-wp-principal.

Reply:
@bsatarnejad You don't wan't to add different classes depending on the used resources in this component. The autocompleter should always behave the same, no matter whether kind of resource is shown. That is why the generic name `op-autocompleter--option-principal` would be best.